scripts/size-estimator.js

```diff
@@ -0,0 +1,11 @@
+// @noflow
+const { execSync }  = require('child_process')
+const packageJson = require('../package.json')
+
+// we estimate redux size based on the content length of the minified umd build hosted by unpkg. This script is brittle but works.
+let reduxSize = execSync("curl -sIL https://unpkg.com/redux/dist/redux.min.js | grep -i Content-Length | tail -1 | awk '{print $2}'").toString()
```

Comment:
you could just make a bundle with `redux` as external and just check this size locally without network requests